docs/solutions/connect_vcns_using_multiple_vnics/vcn2.tf

```diff
@@ -0,0 +1,144 @@
+# The Second VCN
+variable "vcn_cidr2" {
+    default = "10.1.0.0/16"
+}
+
+variable "mgmt_subnet_cidr2" {
+    default = "10.1.0.0/24"
+}
+
+variable "private_subnet_cidr2" {
+    default = "10.1.1.0/24"
+}
+
+# Second VCN
+resource "oci_core_virtual_network" "CoreVCN2" {
+    cidr_block = "${var.vcn_cidr2}"
+    compartment_id = "${var.compartment_ocid}"
+    display_name = "VCN-2"
+}
+
+resource "oci_core_security_list" "MgmtSecurityList2" {
+    compartment_id = "${var.compartment_ocid}"
+    display_name = "MgmtSecurityList2"
+    vcn_id = "${oci_core_virtual_network.CoreVCN2.id}"
+
+    egress_security_rules = [{
+        protocol = "all"
+        destination = "0.0.0.0/0"
+    }]
+
+    ingress_security_rules = [{
```

Comment:
Any reason to allow 80/443 in from anywhere?

Reply:
Removed from both vcn1 and vcn2.tf